web/src/upload.ts

```diff
@@ -283,17 +336,18 @@ export function setup_upload(config: Config): Uppy {
         // allow resuming the upload / avoiding a repeat upload in
         // future browser sessions.
         //
-        // This is not a feature we need across browser sessions. Since these local storage
-        // entries are never garbage-collected, can be accessed via
-        // the browser console even after logging out, and contain
-        // some metadata about previously uploaded files, which seems
-        // like a security risk for using Zulip on a public computer. So we
-        // disable the feature.
+        // This is not a feature we need across browser sessions. Since
```

Comment:
This part of the comment is now confusing and misleading.  When it was right before a line disabling a feature, saying "this is not a feature we need" make sense.  When it's before `urlStorage: new InMemoryUrlStorage(),` it's not as sensical.

You can combine these two paragraphs into one, which just explains the security goal of InMemoryUrlStorage.

Reply:
Attempted a fix at https://github.com/zulip/zulip/compare/31f0f1174dc8d6c6a512cac585e61a072238147e..f1b41f4859ff4ab18f8b2aa2bc9a02e63a886e89

Reply:
Looks good!